From: Thorsten Behrens <tbehrens@suse.com>
Date: Thu, 5 Jun 2014 00:22:40 +0200
Subject: [PATCH] Drop ceph keys into install

Merge distro patches to git.

This was remove-ceph-drop.diff

(cherry picked from commit fab58b206e9244acbd2253194f7ec6b4227a6d04)
---
 Makefile.am | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index cba3af2..806d12d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -24,6 +24,13 @@ install-data-local:
 	-install -m 644 share/id_dsa_drop.ceph.com $(DESTDIR)$(datadir)/ceph/id_dsa_drop.ceph.com
 	-install -m 644 share/id_dsa_drop.ceph.com.pub $(DESTDIR)$(datadir)/ceph/id_dsa_drop.ceph.com.pub
 
+# why is it so hard to make autotools to this?
+install-data-local:
+	-mkdir -p $(DESTDIR)$(datadir)/ceph
+	-install -m 644 share/known_hosts_drop.ceph.com $(DESTDIR)$(datadir)/ceph/known_hosts_drop.ceph.com
+	-install -m 644 share/known_hosts_drop.ceph.com $(DESTDIR)$(datadir)/ceph/id_dsa_drop.ceph.com
+	-install -m 644 share/known_hosts_drop.ceph.com $(DESTDIR)$(datadir)/ceph/id_dsa_drop.ceph.com.pub
+
 all-local:
 if WITH_DEBUG
 #	We need gtest to build the rados-api tests. We only build those in
